apps/devhub/templates/devhub/addons/listing/delete_form.html

```diff
@@ -21,10 +21,10 @@
   {% endif %}
   <p>
     <label>
-      {{ _('Enter your password to confirm your decision:') }}
-      {# see bug 630126 for why this is called 'password-timestamp' #}
-      <input type="password" name="password-{{ timestamp }}" data-name="password" autocomplete="off" required>
```

Comment:
So, it seems the `{{ timestamp }}` hack was added in https://github.com/gkoberger/zamboni/commit/7cd02ca0cfdd333d752b1d105d32329febb599ea. Could we get rid of it now? There's the `timestamp` in the context, and some js code that might now be unused? (Maybe we're using the hack elsewhere too though).